### DISCUSSION TITLE
ERROR resolve image config for docker.io/docker/dockerfile:1

### DISCUSSION THREAD

**Discussion by HiCoderMonkey** [AbdBarho/stable-diffusion-webui-docker#382] [Q&A]
failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to solve with frontend gateway.v0: rpc error: code = Unknown desc = error getting credentials - err: exit status 1, out: `error getting credentials - err: docker-credential-wincred.exe resolves to executable in current directory (.\docker-credential-wincred.exe), out: ```

**Comment by AbdBarho** [comment]
what is the output of `docker pull docker/dockerfile:1`?
